content/applications/inventory_and_mrp/inventory/product_management/product_tracking/create_sn.rst

```diff
@@ -0,0 +1,193 @@
+
+.. _inventory/product_management/detailed-operations-popup:
+
+.. figure:: create_sn/assign-serial-numbers.png
+   :align: center
+   :alt: Display the Detailed Operations smart button and bulleted list icon on a receipt.
+
+   Display the **Detailed Operations** smart button and bulleted list icon on a receipt.
+
+Configuration
+=============
+
+To assign serial numbers to products, activate the :guilabel:`Lots and Serial Numbers` feature in
+:menuselection:`Inventory app --> Configuration --> Settings`, and enable :guilabel:`Tracking by
+Unique Serial Number` in the :guilabel:`Inventory` tab of the item's product form.
+
+.. seealso::
+   - :ref:`Enable serial numbers <inventory/product_management/enable-lots>`
+   - :ref:`Track products by serial numbers <inventory/product_management/configure-lots>`
+
+.. _inventory/product_management/configure-new-serials:
+
+Next, enable creating new serial numbers by going to :menuselection:`Inventory app --> Configuration
+--> Operations Types`.
+
+Select the desired operation type (e.g. *Receipts*, *Delivery Orders* or *Manufacturing*) and select
+the :guilabel:`Create New` option in the :guilabel:`Lots/Serial Numbers` section of the operation
+type's configuration page.
+
+.. image:: create_sn/create-new.png
+   :align: center
+   :alt: Show "Create New" option is selected on the Receipts operation type.
+
+.. _inventory/product_management/detailed-operations:
+
+Detailed Operations
+===================
+
+Serial numbers can be assigned to products when entering stock for the first time, from the
+:guilabel:`Detailed Operations` page on the receipt.
+
+.. important::
+   Serial numbers can **not** be assigned to products on a request for quotation or purchase order —
+   **only** on a receipt.
+
+   .. figure:: create_sn/purchase-order-or-receipt.png
+      :align: center
+      :alt: Show the purchase order and the Receipt smart button
+
+      Screenshot of a "Purchase Order", with the "Receipt" smart button at the top.
+
+   To record an item's serial number before receiving the item, follow the steps in the next
+   sections to assign serial numbers, but do **not** click the receipt's :guilabel:`Validate` button
+   until the products are received from the vendor.
+
+Assign a single serial number to a product by clicking the :guilabel:`Detailed Operations` smart
+button on a receipt.
+
+In the :guilabel:`Lot/Serial Number Name` field, manually type in the serial number for a single
+product.
+
+.. image:: create_sn/add-a-line.png
+   :align: center
+   :alt: Add a line on the Detailed Operations page to assign serial numbers.
+
+When finished, click the receipt's breadcrumbs, and the assigned serial numbers are automatically
+saved.
+
+.. _inventory/product_management/assign-sn:
+
+Assign serial numbers
+=====================
+
+To generate new serial numbers in a sequence, click the :guilabel:`+ (plus)` icon in the
+:ref:`product line <inventory/product_management/detailed-operations-popup>`.
```

Comment:
```suggestion
:ref:`product line <inventory/product_management/detailed-operations-popup>` of a receipt form.
```

Reply:
a screenshot showing this specific icon on the product line could be useful

Reply:
Gonna star this comment, so I add this in an update PR